Simplified.xcodeproj/project.pbxproj

```diff
@@ -2527,7 +2527,6 @@
 				73579046276BE7F2009F1ADF /* NYPLUtilities in Frameworks */,
 				732F046D2633783B0018A82E /* PDFRendererProvider.xcframework in Frameworks */,
 				7347B4E5265EFD3C00E4E386 /* PromisesObjC.xcframework in Frameworks */,
-				731465202786106900027CFA /* PureLayout in Frameworks */,
```

Comment:
Why are we removing this?

Reply:
it was a duplicate, not sure how it got added twice